src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -3101,6 +3104,11 @@ public static InstrumentOption fromString(String value) {
     }
   }
 
+  public enum ChunkOutputType {
```

Comment:
Could you add JavaDoc to explain what this enum is for and what its values mean?

Reply:
Added